packages/block-editor/src/store/test/reducer.js

```diff
@@ -31,6 +35,12 @@ import {
 	lastBlockAttributesChange,
 } from '../reducer';
 
+const blockSelection = combineReducers( {
+	start: selectionStart,
+	end: selectionEnd,
+	initialPosition,
+} );
```

Comment:
This feels a little like cheating. :)

Reply:
Yes, it's to avoid having to rewrite all the tests and ensure it still all works as before.

Reply:
I know, I know. :) But we should _probably_ tackle it head-on, i.e. rewrite the tests. Whether you'd leave that for another PR is up to you!

Reply:
I don't mind rewriting the tests here, I just thought it might make the diff larger and harder to review. If it doesn't sound like it would make things more complicated, I'll make the changes.

Reply:
I'll just rewrite them right now. :)

Reply:
Ok, all tests have been rewritten, I also added some new ones.